### PR TITLE
Introduces buttons

### PR DESCRIPTION
**Pull request by KaiVolland**
This PR introduces the buttons as our first components.
Also fixes the package build.
Adjusts eslint rules.
Adjust babel settings.

### CODE REVIEW

**Comment by KaiVolland** [comment]
@ahennr @dnlkoch @marcjansen
plz review

File: .eslintrc [hunk 1]
```diff
@@ -14,16 +14,30 @@
   },
   "parser": "babel-eslint",
   "globals": {
-    "ol": false
+    "ol": false,
+    "Promise": false,
```
Comment (dnlkoch) [inline .eslintrc]: Minor: Extra comma.

File: src/Button/SimpleButton/SimpleButton.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  /**
+   * Wraps the component.
+   *
+   * @return {Object} The wrapped component.
+   */
+  const setup = () => {
+    const wrapper = mount(<SimpleButton />);
+    return wrapper;
+  };
```
Comment (dnlkoch) [inline src/Button/SimpleButton/SimpleButton.spec.jsx]: Can be replaced with `TestUtils.mount()`.

File: src/Button/ToggleButton/ToggleButton.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+  /**
+   * Wraps the component.
+   *
+   * @return {Object} The wrapped component.
+   */
+  const setup = (props, context) => {
+    const wrapper = mount(<ToggleButton {...props} />, {context});
+    return wrapper;
+  };
```
Comment (dnlkoch) [inline src/Button/ToggleButton/ToggleButton.spec.jsx]: Can be replaced with `TestUtils.mount()`.

File: src/Button/ToggleGroup/ToggleGroup.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+  /**
+   * Wraps the component.
+   *
+   * @return {Object} The wrapped component.
+   */
+  const setup = (props, context) => {
+    const wrapper = mount(<ToggleGroup {...props} />, {context});
+    return wrapper;
+  };
```
Comment (dnlkoch) [inline src/Button/ToggleGroup/ToggleGroup.spec.jsx]: Can be replaced with `TestUtils.mount()`.

File: example-templates/basic.html [hunk 5]
```diff
@@ -10,6 +10,15 @@
...
   <script>/*global hljs*/ hljs.initHighlightingOnLoad();</script>
 </head>
+<style>
+.example-block {
+  padding: 20px;
+}
+.example-block > span {
+  margin-right: 10px;
+}
+</style>
```
Comment (marcjansen) [inline example-templates/basic.html]: :+1: :100:

File: package.json [hunk 6]
```diff
@@ -53,15 +54,17 @@
     "build:examples": "rimraf ./build/examples/* && node tasks/build-examples.js && webpack --config webpack.examples.config.js",
     "build:docs": "jsdoc --package ./package.json --readme ./README.md -c .jsdoc",
     "build:js": "webpack --config webpack.development.config.js && webpack -p --config webpack.production.config.js",
-    "build": "npm run test && npm run build:examples && npm run build:docs && npm run build:js"
+    "build": "npm run clean && BABEL_ENV=build babel src -d dist"
```
Comment (marcjansen) [inline package.json]: Are you sure this is the right thing?

File: package.json [hunk 7]
```diff
@@ -53,15 +54,17 @@
...
   },
   "dependencies": {
     "antd": "2.12.8",
     "lodash": "4.17.4",
     "loglevel": "1.4.1",
     "ol": "4.3.2",
+    "prop-types": "15.5.10",
```
Comment (marcjansen) [inline package.json]: This migfht be one of the causes of the merge conflict

Comment (KaiVolland) [reply package.json]: I don't get why adding a line causes conflicts but you were right.

**Pull request merged**